Singularity/Singularity/Game1.cs

```diff
@@ -91,10 +91,10 @@ protected override void LoadContent()
             // on top of it.
             mScreenManager.AddScreen(mGameScreen);
             mScreenManager.AddScreen(mUserInterfaceScreen);
-            mScreenManager.AddScreen(mMainMenuManager);
+            // mScreenManager.AddScreen(mMainMenuManager); // TODO: This makes it so that the main menu is bypassed
```

Comment:
i think you should put the mainmenu manager back in?

Reply:
I think leaving it out will be good for now since it makes it faster to debug and we're working mostly on the game itself not the menu anymore